ush/forecast_postdet.sh

```diff
@@ -350,27 +349,44 @@ WW3_postdet() {
   fi
 
   echo "Copying WW3 restarts for 'RUN=${RUN}' at '${restart_date}' from '${restart_dir}'"
-  ww3_restart_file="${restart_dir}/${restart_date:0:8}.${restart_date:8:2}0000.restart.ww3"
-  if [[ -s "${ww3_restart_file}" ]]; then
-    ${NCP} "${ww3_restart_file}" "${DATA}/restart.ww3" \
-      || ( echo "FATAL ERROR: Unable to copy WW3 IC, ABORT!"; exit 1 )
-    first_ww3_restart_out=$(date --utc -d "${restart_date:0:8} ${restart_date:8:2} + ${restart_interval} hours" +%Y%m%d%H)
+
+  #First check to see if netcdf restart exists:
```

Comment:
@aerorahul updated the logic based on your suggestions - hopefully this is more clear now.  If not, we'll iterate more!

Reply:
Thanks.  This is much cleaner.
I have a few suggestions below to make the distinction between binary and netcdf restarts explicit.  Please consider.

Reply:
Added! Thanks for making things even clearer!